pal/minigame/taobao.ts

```diff
@@ -93,16 +93,17 @@ minigame.stopAccelerometer = function (res: any) {
 // #endregion Accelerometer
 
 // #region SafeArea
+// It should be a value that is not multiplied by dpr
 minigame.getSafeArea = function () {
```

Comment:
Why not multiply by dpr. As @PPpro said, the app only care about pixel size.

Reply:
this function should return a value that is not multiplied by pdr, beacuse `PAL` will multiply by dpr. We will fix it later.